source/_integrations/brother.markdown

```diff
@@ -26,3 +26,9 @@ To add `Brother Printer` to your installation, go to **Configuration** >> **Inte
 Some very old Brother printers use different data format and these models are not supported. The integration will show information about that during configuration.
 
 </div>
+
+## Configuring the printer
+
+SNMP settings are commonly under Network / Protocol / SNMP.
```

Comment:
I think it might help if we mention that this is on the interface of the device somewhere?
The reason for this, is that this is the Home Assistant documentation and one might start looking for this in the Home Assistant configuration panels.

Reply:
Yeah that makes sense, let me add that.
Is it OK to add the image I used in the pullreq to better illustrate where this setting is?